packaging/language/bower.py

```diff
@@ -37,6 +37,12 @@
     required: false
     default: no
     choices: [ "yes", "no" ]
+  production:
```

Comment:
What should I set `version_added` to? Is there a reference somewhere showing what the next expected release number will be?